### PR TITLE
Updating template files to add quotes

### PR DESCRIPTION
**Pull request by shalakadharap**
# Problem Statement
- Adding quotes to template

# Design
-  Added quotes to template

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [x] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [x] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/utils/setup/openldap/templates/openldap.config.tmpl.1-node [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+        user: "TMPL_ROOT_USER"
+        secret: "TMPL_ROOT_SECRET_KEY"
 cluster:
   TMPL_CLUSTER_ID:
     site:
-      storage_set_count: TMPL_STORAGE_SET_COUNT
+      storage_set_count: "TMPL_STORAGE_SET_COUNT"
     storage_set:
-    - server_nodes: "[TMPL_MACHINE_ID]"
+    - server_nodes: "['TMPL_MACHINE_ID']"
```
Comment (sachinpunadikar) [inline py-utils/src/utils/setup/openldap/templates/openldap.config.tmpl.1-node:18]: This is the only required change, rest all should be reverted back for this file

File: py-utils/src/utils/setup/openldap/templates/openldap.config.tmpl.3-node [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+        user: "TMPL_ROOT_USER"
+        secret: "TMPL_ROOT_SECRET_KEY"
 cluster:
   TMPL_CLUSTER_ID:
     site:
-      storage_set_count: TMPL_STORAGE_SET_COUNT
+      storage_set_count: "TMPL_STORAGE_SET_COUNT"
     storage_set:
-    - server_nodes: "[TMPL_MACHINE_ID1,TMPL_MACHINE_ID2,TMPL_MACHINE_ID3]"
+    - server_nodes: "['TMPL_MACHINE_ID1','TMPL_MACHINE_ID2','TMPL_MACHINE_ID3']"
```
Comment (sachinpunadikar) [inline py-utils/src/utils/setup/openldap/templates/openldap.config.tmpl.3-node:24]: This is the only required change, rest all should be reverted back for this file

File: py-utils/src/utils/setup/openldap/templates/openldap.init.tmpl.1-node [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+        user: "TMPL_ROOT_USER"
+        secret: "TMPL_ROOT_SECRET_KEY"
 cluster:
   TMPL_CLUSTER_ID:
     site:
-      storage_set_count: TMPL_STORAGE_SET_COUNT
+      storage_set_count: "TMPL_STORAGE_SET_COUNT"
     storage_set:
-    - server_nodes: "[TMPL_MACHINE_ID]"
+    - server_nodes: "['TMPL_MACHINE_ID']"
```
Comment (sachinpunadikar) [inline py-utils/src/utils/setup/openldap/templates/openldap.init.tmpl.1-node:18]: This is the only required change, rest all should be reverted back for this file

File: py-utils/src/utils/setup/openldap/templates/openldap.init.tmpl.3-node [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
+        user: "TMPL_ROOT_USER"
+        secret: "TMPL_ROOT_SECRET_KEY"
 cluster:
   TMPL_CLUSTER_ID:
     site:
-      storage_set_count: TMPL_STORAGE_SET_COUNT
+      storage_set_count: "TMPL_STORAGE_SET_COUNT"
     storage_set:
-    - server_nodes: "[TMPL_MACHINE_ID1,TMPL_MACHINE_ID2,TMPL_MACHINE_ID3]"
+    - server_nodes: "['TMPL_MACHINE_ID1','TMPL_MACHINE_ID2','TMPL_MACHINE_ID3']"
```
Comment (sachinpunadikar) [inline py-utils/src/utils/setup/openldap/templates/openldap.init.tmpl.3-node:24]: This is the only required change, rest all should be reverted back for this file

File: py-utils/src/utils/setup/openldap/templates/openldap.test.tmpl.3-node [hunk 5]
```diff
@@ -1,24 +1,24 @@
...
+        user: "TMPL_ROOT_USER"
+        secret: "TMPL_ROOT_SECRET_KEY"
 cluster:
   TMPL_CLUSTER_ID:
     site:
-      storage_set_count: TMPL_STORAGE_SET_COUNT
+      storage_set_count: "TMPL_STORAGE_SET_COUNT"
     storage_set:
-    - server_nodes: "[TMPL_MACHINE_ID1,TMPL_MACHINE_ID2,TMPL_MACHINE_ID3]"
+    - server_nodes: "['TMPL_MACHINE_ID1','TMPL_MACHINE_ID2','TMPL_MACHINE_ID3']"
```
Comment (sachinpunadikar) [inline py-utils/src/utils/setup/openldap/templates/openldap.test.tmpl.3-node:24]: This is the only required change, rest all should be reverted back for this file

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Work on comments. If you need any clarification reach to @gauravchaudhari02

**Comment by shalakadharap** [comment]
Duplicate

**Pull request closed without merging**